README.md

```diff
@@ -87,6 +87,7 @@ We have the following branches
 
  * **gh-pages**
    Hosting the landing page [phimp.me](http://phimp.me)  
```

Comment:
Please remove this line entirely. Just keep the new one and remove the word currently from it.

Reply:
ohk... working on it 👍